src/java.base/share/classes/java/util/TreeMap.java

```diff
@@ -420,10 +420,10 @@ else if (cmp > 0)
     }
 
     /**
-     * Gets the entry corresponding to the specified key; if no such entry
-     * exists, returns the entry for the least key greater than the specified
-     * key; if no such entry exists (i.e., the greatest key in the Tree is less
-     * than the specified key), returns {@code null}.
+     * Gets the entry corresponding to the specified key; returns the entry for
+     * the least key greater than the specified key; if no such entry exists
```

Comment:
Can we change this to: (same for the floor one)
```
Returns the entry for the least key greater than or equal to the specified key;
if no such entry exists (i.e. the specified key is greater than any key in the tree),
returns {@code null}.
```
What do you think of my version?

Reply:
Looks good to me and is similar to other methods of the form `get*` such as `getHigherEntry`. I could make that change and ensure documentation is similar for methods.

Reply:
Indeed, ceil works like higher; given a key `k`, ceil allows exact `k` while higher does not, and they are otherwise the same. Same for floor/lower.